capsules/src/lib.rs

```diff
@@ -1,4 +1,5 @@
-#![feature(const_fn)]
+#![feature(const_fn_trait_bound)]
+#![feature(const_fn_unsize)]
```

Comment:
`const_fn_unsize` was stabilized two days ago, but I think its fine to stick with the slightly older nightly for now and remove this next time we update.

Reply:
Maybe we can give this one until Friday and see if Rust can get us a nightly with everything working before then?

Reply:
That's fine too